salt/docker/init.sls

```diff
@@ -83,13 +83,18 @@ docker:
     - onlyif: systemctl status docker.service
     - onchanges:
       - /etc/systemd/system/docker.service.d/proxy.conf
```

Comment:
I wonder if we can also use `reload` when `proxy.conf` changes, and we can basically use the same state for any change on the docker configuration.

Reply:
we need to do a daemon-reload and a restart, see: https://docs.docker.com/config/daemon/systemd/#httphttps-proxy

updated the code